web/frontend/src/components/slam/index.svelte

```diff
@@ -278,13 +277,21 @@ const toggleAxes = () => {
 
 const handleMoveClick = async () => {
   try {
+    // set pose in frame
+    const lastPose = await slamClient.getPosition();
+    const destinationMM = lastPose.pose!;
+    destinationMM.x = destination!.x * 1000;
+    destinationMM.y = destination!.y * 1000;
     const base = bases[0]!;
```

Comment:
[minor stuff] Could we change this to be a little more defensive? In general, we should avoid using the assertion operator `!` (in App we have linters against this) because it's unsafe. Maybe we could use the nullish coalescing operator `??` to provide a reasonable default? Or return and display an error?